source/release-notes/1.7.txt

```diff
@@ -0,0 +1,43 @@
+.. _c2c-release-notes-1.7:
+
+===============================
+Release Notes for mongosync 1.7
+===============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _1.7.0-c2c-release-notes:
+
+1.7.0 Release
+-------------
+
+**Upcoming**
+
+Sync Multiple Source Clusters Simultaneously with a Destination Cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/many-with-one-cluster.rst
+
+For an example, see :ref:`c2c-quickstart-many-with-one`. For
+limitations, see :ref:`c2c-limitations_multiple_sync`.
+
+Set a Migration Name
+~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/migrationName-description.rst
+
+To set a migration name, see:
+
+- :setting:`migrationName` configuration file setting
+- :option:`--migrationName` command line option
+
+Minimum Supported Version
+-------------------------
+
+In 1.7.0, the minimum supported version of MongoDB is 6.0.8.
```

Comment:
**Suggestion**
A bit off topic and not for this ticket, but would it be easier to manage all the support version in a single file and include them at the bottom similar to how we do platform support for server:

https://www.mongodb.com/docs/manual/administration/production-notes/#platform-support-matrix

Reply:
As written, it follows the precedent set for previous C2C release notes. I know there is a project under investigation to consolidate support matrices into one file.